Vagrantfile

```diff
@@ -0,0 +1,13 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/xenial64"
+  # config.vm.network "public_network", :bridge => "eth0", ip: "xxx.yyy.zzz.153", :netmask => "255.255.255.128", auto_config: false
+  config.vm.network "public_network", bridge: "enp0s31f6"
```

Comment:
Vagrant is very specific about what it wants in the network section of provisioning. See: https://www.vagrantup.com/docs/networking/public_network.html#default-network-interface

When you do `vagrant up`, the host machine checks to see if it has an interface specified in `config.vm.network`, and if it has one that matches that string, then vagrant will proceed forward. If the host doesn't have a matching interface, vagrant will ask the user which interface on their host machine they would like to forward via bridging to the VM.

The reason I put in `enp0s31f6` is that that is the interface that is on our Jenkins machine. Since there is no attachable TTY in Jenkins, If I put any other string there, Jenkins will fail as vagrant is waiting for user input as to which interface to select

Reply:
Perhaps we could keep the host-specific networking options out of the committed Vagrantfile and instead [provide these as defaults](https://www.vagrantup.com/docs/vagrantfile/#load-order-and-merging) in `~/.vagrant.d`

Reply:
That's frustrating, but there could be a workaround.

We can create and install our own udev rules as a part of provisioning the VM, which should allow us to give a name to the default adapter.

Reply:
No longer need a bridged interface for system tests (at least so far).
Using a private network with a vagrant DHCP server for system tests